sections/semantics-interactive-elements.include

```diff
@@ -386,67 +375,25 @@
   The <dfn attribute for="HTMLMenuElement"><code>label</code></dfn> IDL attribute must
   <a>reflect</a> the content attribute of the same name.
 
-  </div>
-
-  <div class="example">
-    In this example, the <{menu}> element is used to describe a toolbar with three menu
-    buttons on it, each of which has a dropdown menu with a series of options:
-
-  <pre highlight="html">
-&lt;menu&gt;
-  &lt;li&gt;
-  &lt;button type=menu value="File" menu="filemenu"&gt;
-  &lt;menu id="filemenu" type="context"&gt;
-    &lt;menuitem onclick="fnew()" label="New..."&gt;
-    &lt;menuitem onclick="fopen()" label="Open..."&gt;
-    &lt;menuitem onclick="fsave()" label="Save"&gt;
-    &lt;menuitem onclick="fsaveas()" label="Save as..."&gt;
-  &lt;/menu&gt;
-  &lt;/li&gt;
-  &lt;li&gt;
-  &lt;button type=menu value="Edit" menu="editmenu"&gt;
-  &lt;menu id="editmenu" type="context"&gt;
-    &lt;menuitem onclick="ecopy()" label="Copy"&gt;
-    &lt;menuitem onclick="ecut()" label="Cut"&gt;
-    &lt;menuitem onclick="epaste()" label="Paste"&gt;
-  &lt;/menu&gt;
-  &lt;/li&gt;
-  &lt;li&gt;
-  &lt;button type=menu value="Help" menu="helpmenu"&gt;
-  &lt;menu id="helpmenu" type="context"&gt;
-    &lt;menuitem onclick="location='help.html'" label="Help"&gt;
-    &lt;menuitem onclick="location='about.html'" label="About"&gt;
-  &lt;/menu&gt;
-  &lt;/li&gt;
-&lt;/menu&gt;
-    </pre>
 
-    In a supporting user agent, this might look like this (assuming the user has just activated the
-    second button):
-
-    <img src="images/sample-toolbar-1.png" width="303" height="101" alt="A toolbar with three buttons, labeled 'File', 'Edit', and 'Help'; where if you select the 'Edit' button you get a drop-down menu with three more options, 'Copy', 'Cut', and 'Paste'." />
```

Comment:
Don't forget to remove the image resource as well--this is the only use of `sample-toolbar-1.png` in the source.